packages/framework-core/src/booster-graphql-dispatcher.ts

```diff
@@ -118,6 +127,7 @@ export class BoosterGraphQLDispatcher {
       }
       const resolverContext: GraphQLResolverContext = {
         connectionID: envelope.connectionID,
+        responseHeaders: {},
```

Comment:
I think this is where we should set the default headers. Ideally, we should provide a configuration option to allow users to override them at config time too, so this would be something like:

```suggestion
        responseHeaders: { ...this.config.defaultResponseHeaders },
```

In this way, we can hardcode the default headers (Or an empty object if there are none) and the user can easily set the default response headers for an environment.